src/fhirtypes/common.ts

```diff
@@ -301,8 +301,11 @@ export function replaceReferences<T extends AssignmentRule | CaretValueRule>(
   } else if (value instanceof FshCode) {
     const [system, ...versionParts] = value.system?.split('|') ?? [];
     const version = versionParts.join('|');
-    const codeSystem = tank.fish(system, Type.CodeSystem);
-    const codeSystemMeta = fisher.fishForMetadata(codeSystem?.name, Type.CodeSystem);
+    const codeSystem = tank.fish(system, Type.CodeSystem) ?? tank.fish(system, Type.Instance);
+    const codeSystemMeta =
+      codeSystem instanceof FshCodeSystem
+        ? fisher.fishForMetadata(codeSystem?.name, Type.CodeSystem)
+        : fisher.fishForMetadata(codeSystem?.name, Type.Instance);
```

Comment:
I think this ends up being too permissive in terms of what types of Instances can be used. For example, a CapabilityStatement can have a name and a url, so an Instance of a CapabilityStatement can be used without any errors appearing. So, there probably needs to be a check that that the Instance fished up is actually a CodeSystem.

Reply:
This is an excellent point, addressed in commit `17993d4`.